scripts/tests/helper/initializer.js

```diff
@@ -1,105 +0,0 @@
-const { formatBytes32String } = require("ethers/lib/utils");
```

Comment:
I guess Pan Vitalii wanted to get rid of initializer in test/helper since migrations/deployment script does enough job for initializtion.